Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -700,35 +700,78 @@ class SubprocessPidfdWatcherTests(SubprocessWatcherMixin,
                                       test_utils.TestCase):
         Watcher = unix_events.PidfdChildWatcher
 
-else:
-    # Windows
-    class SubprocessProactorTests(SubprocessMixin, test_utils.TestCase):
 
-        def setUp(self):
-            super().setUp()
-            self.loop = asyncio.ProactorEventLoop()
-            self.set_event_loop(self.loop)
+    class GenericWatcherTests(test_utils.TestCase):
 
+        def test_create_subprocess_fails_with_inactive_watcher(self):
+            watcher = mock.create_autospec(
+                asyncio.AbstractChildWatcher,
+                **{"__enter__.return_value.is_active.return_value": False}
+            )
 
-class GenericWatcherTests:
+            async def execute():
+                asyncio.set_child_watcher(watcher)
 
-    def test_create_subprocess_fails_with_inactive_watcher(self):
+                with self.assertRaises(RuntimeError):
+                    await subprocess.create_subprocess_exec(
+                        os_helper.FakePath(sys.executable), '-c', 'pass')
 
-        async def execute():
-            watcher = mock.create_authspec(asyncio.AbstractChildWatcher)
-            watcher.is_active.return_value = False
-            asyncio.set_child_watcher(watcher)
+                watcher.add_child_handler.assert_not_called()
 
-            with self.assertRaises(RuntimeError):
-                await subprocess.create_subprocess_exec(
-                    os_helper.FakePath(sys.executable), '-c', 'pass')
+            with asyncio.Runner(loop_factory=asyncio.new_event_loop) as runner:
```

Comment:
`asyncio.new_event_loop` is the default value, so why pass it at all?

Reply:
Ah well about that...

Reply:
This is extremely confusing asyncio behavior https://github.com/python/cpython/pull/94593/files#r914528655